lastseen.py

```diff
@@ -0,0 +1,87 @@
+from itertools import chain
+from datetime import datetime
+from errbot import BotPlugin, botcmd
+
+USAGE_STR = 'Nick not specified. (usage: !last_seen <nick>)'
+CONFIG_TEMPLATE = {
+    # locale-preffered format is default
+    'TIME_FORMAT': '%c',
+    'MAX_CHARS': 40
```

Comment:
@JakubNvk While we understand what `MAX_CHARS` means in this context, it may be quite ambiguous for someone who has not read the code (i.e. I can easily see them asking the end user `MAX_CHARS` of the whole message or `MAX_CHARS` of the message the user sent?).

Can we change it to something more descriptive?